apps/web/src/components/templates/in-app-editor/TemplateInAppEditor.tsx

```diff
@@ -1,25 +1,21 @@
+import { useInputState } from '@mantine/hooks';
+import { FeedEntity } from '@novu/dal';
```

Comment:
dal import here again :) I've added the linter on master already for us

Reply:
Yea the linter will really help, so really thanks for it!!!!